src/Polysemy/Internal/Tactics.hs

```diff
@@ -38,41 +41,48 @@ import Polysemy.Internal.Union
 -- data 'Polysemy.Resource.Resource' m a where
 --   'Polysemy.Resource.Bracket' :: m a -> (a -> m ()) -> (a -> m b) -> 'Polysemy.Resource.Resource' m b
 -- @
---
--- Here we have an @m a@ which clearly needs to be run first, and then
--- subsequently call the @a -> m ()@ and @a -> m b@ arguments. In a 'Tactical'
--- environment, we can write the threading code thusly:
+-- We can interpret this effect like so:
 --
 -- @
--- 'Polysemy.Resource.Bracket' alloc dealloc use -> do
---   alloc'   <- 'runT'  alloc
---   dealloc' <- 'bindT' dealloc
---   use'     <- 'bindT' use
+-- runResource = Polysemy.interpretH $ \\case
+--  'Polysemy.Resource.Bracket' alloc dealloc use -> do
+--    resource <- 'runTSimple' alloc
+--    result <- 'bindTSimple' use resource
+--    'bindTSimple' dealloc resource
+--    pure result
 -- @
 --
--- where
+-- To run embedded higher-order actions (e.g. @alloc :: m a@s) we use 'runTSimple'.
 --
--- @
--- alloc'   ::         'Polysemy.Sem' ('Polysemy.Resource.Resource' ': r) (f a1)
--- dealloc' :: f a1 -> 'Polysemy.Sem' ('Polysemy.Resource.Resource' ': r) (f ())
--- use'     :: f a1 -> 'Polysemy.Sem' ('Polysemy.Resource.Resource' ': r) (f x)
--- @
+-- If we have a kleisli action (e.g. @use :: a -> m b@) we can instead use 'bindTSimple'.
+--
+-- There are however some cases where these functions are not sufficient.
+--
+-- Consider the example of the 'Polysemy.Reader.local' action.
 --
--- The @f@ type here is existential and corresponds to "whatever
--- state the other effects want to keep track of." @f@ is always
--- a 'Functor'.
+-- In it, we want any inner computations to be executed with a __modified__ environment.
 --
--- @alloc'@, @dealloc'@ and @use'@ are now in a form that can be
--- easily consumed by your interpreter. At this point, simply bind
--- them in the desired order and continue on your merry way.
+-- This is not possible with 'runTSimple' and 'bindTSimple', because they automatically
+-- take the care to run all your embedded actions by using the current set of interpreters.
+--
+-- Instead, we need to use 'runT' and 'bindT', which give more control over how inner actions are run.
+--
+-- As an example, let's look at the implementation for running a 'Polysemy.Reader.Reader'
+-- with a constant value:
+--
+-- @
+-- 'Polysemy.Reader.runReader' i = 'Polysemy.interpretH' $ \\case
+--   'Polysemy.Reader.Ask' -> pureT i
+--   'Polysemy.Reader.Local' f m -> do
+--     mm <- 'runT' m
```

Comment:
This `<` needs to be escaped

Reply:
Why? Seems to render find, or am I missing something? I'm not too good at haddock!

Reply:
oh yeah. weird, I remember that being a problem